x-pack/plugins/profiling/server/lib/setup/steps/component_template_profiling_stackframes.json

```diff
@@ -0,0 +1,39 @@
+{
+  "settings": {
+    "index": {
+      "number_of_shards": 16,
+      "refresh_interval": "10s"
+    }
+  },
+  "mappings": {
+    "_source": {
+      "mode": "synthetic"
+    },
+    "properties": {
+      "ecs.version": {
+        "type": "keyword",
+        "index": true
+      },
+      "Stackframe.line.number": {
+        "type": "integer",
+        "index": false
+      },
+      "Stackframe.file.name": {
+        "type": "keyword",
+        "index": false
+      },
+      "Stackframe.source.type": {
+        "type": "short",
+        "index": false
+      },
```

Comment:
Remove this field from the stackframes (see https://github.com/elastic/prodfiler/issues/2944, Point 3)
```suggestion
```